device-types/Supermicro/SYS-6018R-MD.yml

```diff
@@ -0,0 +1,27 @@
+---
+manufacturer: Supermicro
+model: SYS-6018R-MD
+slug: supermicro-sys-6018r-md
+part_number: SYS-6018R-MD
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (26 lbs or 11.80 kg)